File: client/src/app/_models/comicForGenreCusDto.ts
import { ChapterForGenreCusDto } from "./chapterForGenreCusDto";

export interface ComicForGenreCusDto {
  id: number;
  comicName: string;
  isFeatured: boolean;
  mainImage: string;
  rate: number;
  noReviews: number;
  noFollows: number;
  noComments: number;
  chapters: ChapterForGenreCusDto[];
}

File: API/Dtos/ComicForGenreCusDto.cs
namespace API.Dtos
{
    public class ComicForGenreCusDto
    {
        public int Id { get; set; }
        public string ComicName { get; set; }
        public bool IsFeatured { get; set; }
        public string MainImage { get; set; }
        public int Rate { get; set; }
        public int NOReviews { get; set; }
        public int NOFollows { get; set; }
        public int NOComments { get; set; }
        public List<ChapterForGenreCusDto> Chapters { get; set; }
    }
}